src/book/book.rs

```diff
@@ -166,6 +174,31 @@ impl Chapter {
     }
 }
 
+/// The representation of a "virtual chapter", available for namespacing
+/// purposes and not mapping to a file on disk.
+#[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
+pub struct VirtualChapter {
+    /// The chapter's name.
+    pub name: String,
+    /// The chapter's contents.
+    pub content: String,
```

Comment:
Is there any content in a virtual chapter?

Reply:
I don't believe so. A virtual chapter just has a name, maybe a number, and a bunch of sub-items.

Reply:
> Is there any content in a virtual chapter?

This was actually a question somewhere else in a `// TODO`. I will remove the field.